deployment/terraform/cdn.tf

```diff
@@ -0,0 +1,62 @@
+resource "aws_cloudfront_distribution" "cdn" {
+  origin {
+    domain_name = "${aws_route53_record.origin.fqdn}"
+    origin_id   = "originPotsdamSite"
+
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "https-only"
+      origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
+    }
+  }
+
+  enabled          = true
+  http_version     = "http2"
+  comment          = "Geotrellis Potsdam Demo (${var.environment})"
```

Comment:
s/Geotrellis/GeoTrellis/

Also, it might make sense to omit that altogether because we're already within that account.